Dalamud/Interface/Components/ImGuiComponents.DisabledButton.cs

```diff
@@ -21,15 +23,17 @@ public static partial class ImGuiComponents
     /// <returns>Indicator if button is clicked.</returns>
     public static bool DisabledButton(FontAwesomeIcon icon, int? id = null, Vector4? defaultColor = null, Vector4? activeColor = null, Vector4? hoveredColor = null, float alphaMult = .5f)
     {
-        ImGui.PushFont(UiBuilder.IconFont);
-
-        var text = icon.ToIconString();
-        if (id.HasValue)
-            text = $"{text}##{id}";
-
-        var button = DisabledButton(text, defaultColor, activeColor, hoveredColor, alphaMult);
-
-        ImGui.PopFont();
+        bool button;
+        using (ImRaii.PushFont(UiBuilder.IconFont))
+        {
+            var text = icon.ToIconString();
+            if (id.HasValue)
+            {
+                text = $"{text}##{id}";
+            }
+
+            button = DisabledButton(text, defaultColor, activeColor, hoveredColor, alphaMult);
```

Comment:
You can just return here, no need to store it just for return

Return ends the scope automatically